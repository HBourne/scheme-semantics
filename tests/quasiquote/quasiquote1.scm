[define x 0]
[define y [lambda[] `[,[set! x [+ x 1]] 3 ]]]
[display [y]]
  [newline]
[display x]
  [newline]
[display [y]]
  [newline]
[display x]
  [newline]
