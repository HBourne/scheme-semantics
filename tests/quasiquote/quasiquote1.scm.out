(#<void> 3)
1
(#<void> 3)
2
